src/config.ts

```diff
@@ -221,11 +221,17 @@ export class KubeConfig {
                 },
             },
         ];
+        const namespaceFile = `${pathPrefix}${Config.SERVICEACCOUNT_NAMESPACE_PATH}`;
+        let namespace: string | undefined;
+        if (fileExists(namespaceFile)) {
+            namespace = fs.readFileSync(namespaceFile, 'utf8')
```

Comment:
missing a semi-colon.

Reply:
Sorry I miss the notification.
I just push it.
Thx for the review